howto/mro.rst:13: [timeout] https://www.phyast.pitt.edu/~micheles/: HTTPSConnectionPool(host='www.phyast.pitt.edu', port=443): Max retries exceeded with url: /~micheles/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f92ef129820>, 'Connection to www.phyast.pitt.edu timed out. (connect timeout=30)'))
library/unittest.rst:67: [timeout] http://lists.idyll.org/listinfo/testing-in-python: HTTPConnectionPool(host='lists.idyll.org', port=80): Max retries exceeded with url: /listinfo/testing-in-python (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f92ef3f6c90>, 'Connection to lists.idyll.org timed out. (connect timeout=30)'))
whatsnew/3.2.rst:978: [timeout] https://osl.cs.illinois.edu/media/papers/karmani-2009-barrier_synchronization_pattern.pdf: HTTPSConnectionPool(host='osl.cs.illinois.edu', port=443): Max retries exceeded with url: /media/papers/karmani-2009-barrier_synchronization_pattern.pdf (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f92ef3f51c0>, 'Connection to osl.cs.illinois.edu timed out. (connect timeout=30)'))
